README.md

```diff
@@ -1 +1,34 @@
-# Review-past-project
\ No newline at end of file
+# Review-past-project
+
+Project for the 2.0 reviews at Microverse
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+
+## Live Demo
+
+![Screenshot](https://github.com/guillainbisimwa/Review-past-project/blob/feature/Screenshot.png)
+
+## Author
+
+👤 **Guillain Bisimwa**
+
+- Github: [@guillainbisimwa](https://github.com/guillainbisimwa)
+- Twitter: [@gullain_bisimwa](https://twitter.com/gullain_bisimwa)
+- Linkedin: [guillain-bisimwa](https://www.linkedin.com/in/guillain-bisimwa-8a8b7a7b/)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+Feel free to check the issues page.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
Your link is broken! Please fix it

Reply:
### Hello Guillian!, I think it would be better if you `kindly` suggest changes to the student and give a little explanation as to why the change is required, for example: 

> Would you kindly fix this broken link? Having a well-documented README gives a really good first impression and shows professionalism.